src/pages/workspaces/workspace/Dashboard/BucketLocation.ts

```diff
@@ -42,7 +42,7 @@ export const BucketLocation = requesterPaysWrapper({ onDismiss: _.noop })((props
         .checkBucketLocation(googleProject, bucketName);
       setBucketLocation(response);
     } catch (error) {
-      if (error != null && typeof error === 'object' && 'requesterPaysError' in error) {
+      if (isBucketErrorRequesterPays(error)) {
```

Comment:
This was actually a bug in that we didn't check that `error.requesteredPaysError` was true… we just checked that the key existed in the object.